src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/addnote/AddNoteAboutToSubmitHandler.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.addnote;
+
+import static java.util.Objects.nonNull;
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+@Component
+@Slf4j
+public class AddNoteAboutToSubmitHandler  implements PreSubmitCallbackHandler<SscsCaseData> {
+
+    protected final IdamClient idamClient;
+
+    @Autowired
+    public AddNoteAboutToSubmitHandler(IdamClient idamClient) {
+        this.idamClient = idamClient;
+    }
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
+        requireNonNull(callback, "callback must not be null");
+        requireNonNull(callbackType, "callbacktype must not be null");
+
+        return callbackType.equals(CallbackType.ABOUT_TO_SUBMIT)
+                && (callback.getEvent() == EventType.ADD_NOTE
+                || callback.getEvent() == EventType.NON_COMPLIANT_SEND_TO_INTERLOC
+                || callback.getEvent() == EventType.ADMIN_SEND_TO_INTERLOCUTORY_REVIEW_STATE
+                || callback.getEvent() == EventType.INTERLOC_SEND_TO_TCW
+                || callback.getEvent() == EventType.TCW_REFER_TO_JUDGE
+                || callback.getEvent() == EventType.SEND_TO_ADMIN
```

Comment:
The event PROCESS_AUDIO_VIDEO also uses notes for SendToJudge and SendToTcw actions